### DISCUSSION TITLE
Feature Request - Tool path odometer

### DISCUSSION THREAD

**Discussion by Jokemeister1** [OrcaSlicer/OrcaSlicer#5269] [Ideas]
I've searched but come up blank.

The possibility to have the tool path distance traveled in metres/feet (a metre is a unit of distance,  a meter is a gauge/gage...rant over  ;-) ) displayed in the preview window under perhaps the "Speed" display since it already has the option for checking the Travel box to show the tool path. Preference would be to show the job total and not just each layer total as it informs users in regards to life expectancy of machine components like belts and motors if they tally up their jobs. Ultimately some way to keep a running tally on printed jobs but that is probably something to be added to the likes of Klipper/Moonraker/Fluidd...whichever makes the most sense.

eg. in this window
![speed](https://github.com/SoftFever/OrcaSlicer/assets/168946869/fffe9adf-268d-4bd2-a31b-4022065f4280)

in a similar way to the Line Type totals
![image](https://github.com/SoftFever/OrcaSlicer/assets/168946869/afa4d2e8-dee4-439d-9e8a-7dd1c61306e0)

**Comment by sscargal** [comment]
I wanted the same feature, so I implemented it and created PR #11187. Hope it helps.

**Comment by Jokemeister1** [comment]
Thank you for your work on this.

Have a great day.
